src/components/FAQs.js

```diff
@@ -0,0 +1,23 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import enhanceCollection from 'phenomic/lib/enhance-collection';
+
+import PagesList from '../components/PagesList';
+
+
+const FAQs = (props, { collection }) => {
+  const faqs = enhanceCollection(collection, {
+    filter: { __filename: /^faq\// },
+    sort: 'title',
+  });
+
+  return (
+    <PagesList pages={faqs} />
+  );
+};
+
+FAQs.contextTypes = {
+  collection: PropTypes.array.isRequired,
```

Comment:
So, in favor of `PropTypes` for this thing rather than doing a bunch of work trying to make it `TypeScript` or just going with nothing?